src/tests/test_utils/common_test_utils/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@ function(add_common_utils ADD_TARGET_NAME)
     )
 
     ov_developer_package_export_targets(TARGET ${ADD_TARGET_NAME}
-                                        INSTALL_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/include/")
+                                        INSTALL_INCLUDE_DIRECTORIES
+                                            "${CMAKE_CURRENT_SOURCE_DIR}/include/"
+                                            "${CMAKE_CURRENT_SOURCE_DIR}/src/"
```

Comment:
Needed to add `src` to get `test_constants.cpp` required for NPU plugin

Reply:
We should not use OV sources directly, why don't you link with test lib instead?

Reply:
Link with `common_test_utils` target in NPU tests instead

Reply:
Hello @praasz @mryzhov, the file test_constants.cpp is used as a part of source code in npu-plugin,
ref: https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/blob/0592450b957320875c737f7cd8c4cc5023d5da58/tests/functional/imd_tests/CMakeLists.txt#L47 
therefore shall we just copy this file to the install package?
Hi @Maxim-Doronin, another choice to avoid using test_constants.cpp is to create a copy of this file in npu-plugin. Shall we consider this solution?

Reply:
These files is util for openvino tests as internal files so it should be use only if targets provided.
As there are only some const strings, the copy inside NPU plugin repo should be OK.
What is used from this file ?

The device names can be copied but NPU is defined by NPU plugin, maybe it is available in this repo.
The other string looks like some internal options and can be specified in this repo if required.

Reply:
A PR is created to get rid of the dependency of test_constants.cpp in NPU-plugin. Let's wait for a while and then check if the problem is fixed.
Ref: https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/19230

Reply:
> A PR is created to get rid of the dependency of test_constants.cpp in NPU-plugin.

The PR has been merged. Reverted this change and undergoing testing.

Reply:
This change has been reverted and testing has passed for NPU build.